### PR TITLE
Register `avengers-sonarqube.is-a-good.dev`

### PR DESCRIPTION
**Pull request by hxhieu**

### CODE REVIEW

File: sub-logs/avengers-sonarqube.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "repo": "https://github.com/hxhieu/is-a-good-dev-register",
+  
+    "owner": {
+      "username": "hxhieu",
+      "email": "hugh.hoang@gmail.com"
+    },
+  
+    "target": {
+      "CNAME": {
+        "name": "www",
```
Comment (Tweak4141) [inline sub-logs/avengers-sonarqube.json:11]: Should be avengers-sonarqube

File: sub-logs/avengers-sonarqube.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        "value": "sonaqube.jollyocean-513cc86d.australiaeast.azurecontainerapps.io"
+      },
+  
+      "TXT": {
+        "name": "asuid.www",
```
Comment (Tweak4141) [inline sub-logs/avengers-sonarqube.json:16]: asuid.avengers-sonarqube

File: sub-logs/avengers-sonarqube.json (hunk 1 above)
Comment (Tweak4141) [inline sub-logs/avengers-sonarqube.json:11]: avengers-sonarqube

**Review by Tweak4141 — CHANGES_REQUESTED** [review]
Name must be subdomain value.

File: sub-logs/avengers-sonarqube.json (hunk 1 above)
Comment (hxhieu) [reply sub-logs/avengers-sonarqube.json]: Thanks and fixed

**Review by Tweak4141 — APPROVED** [review]

**Pull request merged**